### DISCUSSION TITLE
Mojave support???

### DISCUSSION THREAD

**Discussion by tiziodcaio** [chris1111/Legacy-Video-patch#36] [Ideas]
It's possible to have it or not?

**Comment by chris1111** [comment]
Nvidia
https://github.com/chris1111/Fix-Old-NVIDIA-macOS-Mojave

Intel HD 3000
https://github.com/chris1111/Fix-Graphics-HD-3000-Mojave-10.14

**Comment by tiziodcaio** [comment]
Thx
